hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/keyvalue/KvMetadataAvroResolver.java

```diff
@@ -91,18 +91,19 @@ public KvMetadata resolveMetadata(
         }
         maybeAddDefaultField(isKey, resolvedFields, fields, QueryDataType.OBJECT);
 
+        String recordName = options.getOrDefault((isKey ? "key" : "value") + ".record.name", "jet.sql");
```

Comment:
was `key|value.record.name` already supported option or did you just add it?

Reply:
if it is a new and non-standard Kafka option we should document that

Reply:
I just made up a new option to specify record name for `[Topic]RecordNameStrategy`.